src/pages/cody.tsx

```diff
@@ -103,60 +184,355 @@ const CodyPage: FunctionComponent = () => {
-                }
-                tabContent={IMAGE_TAB_CONTENT}
-                isLight={true}
-            />
-
-            <CodyAutocomplete isLight={true} wrapperClassName="z-[20] md:z-0 !mt-24" />
+            <ContentSection parentClassName="py-16 md:!py-28">
+                <CodyTwoColumnSection
+                    subTitle="The best models"
+                    title="Cody uses the latest AI models. Never settle for last-gen."
+                    description="Cody gives you access to the best and latest LLMs. Choose the best one for your use
+                    cases, optimized for speed or power."
+                    extraContent={
+                        <>
+                            <div>
+                                <div className="mb-4 mt-10 text-base font-medium tracking-tight md:mt-[13px]">
+                                    Optimized for Power
+                                </div>
+                                <div className="mb-10 flex max-w-[556px] flex-row flex-wrap gap-4">
+                                    {optimizedPoweritems.map(item => (
+                                        <div key={item.label} className="flex flex-row gap-x-1">
+                                            <IDEBadge
+                                                text={item.label}
+                                                className="w-fit !rounded-[10px] !border !border-[#E4E9F4] !bg-gray-100 !px-3 !py-1 !font-sans !text-sm !font-normal  !text-black"
+                                                src={item.iconUrl}
+                                            />
+                                        </div>
+                                    ))}
+                                </div>
+                            </div>
+                            <div>
+                                <div className="mb-4 text-base font-medium">Optimized for Speed</div>
+                                <div className="flex max-w-[556px] flex-row flex-wrap gap-4">
+                                    {optimizedSpeed.map(item => (
+                                        <div key={item.label} className="flex flex-row gap-x-1">
+                                            <IDEBadge
+                                                text={item.label}
+                                                className="w-fit !rounded-[10px] !border !border-[#E4E9F4] !bg-gray-100 !px-3 !py-1 !font-sans !text-sm !font-normal !text-black"
+                                                src={item.iconUrl}
+                                            />
+                                        </div>
+                                    ))}
+                                </div>
+                            </div>
+                        </>
+                    }
+                    imgSrc="/assets/cody/placeholder-cody-models.svg"
+                />
+            </ContentSection>
+            <ContentSection parentClassName="!py-0">
```

Comment:
This was my mistake, there should be 16 (64px) space below the "Rob Linger, AI Software...." content on this quote block. 
<img width="903" alt="Screenshot 2024-09-25 at 8 47 55 AM" src="https://github.com/user-attachments/assets/a49335da-1a6c-4cd7-82eb-74765b1ecfd5">